deployment/instascale-clusterrole.yaml

```diff
@@ -50,3 +50,10 @@ rules:
   - update 
   - delete
   - patch
+
+- apiGroups:
```

Comment:
We will also need to update the deployment in the codeflare-operator. https://github.com/project-codeflare/codeflare-operator/blob/main/config/internal/instascale/clusterrole.yaml.tmpl

Reply:
Will update here also. Thanks